src/test/javascript/spec/component/lecture/lecture-attachments.component.spec.ts

```diff
@@ -80,7 +82,7 @@ describe('LectureAttachmentsComponent', () => {
     beforeEach(async () => {
         return TestBed.configureTestingModule({
             imports: [TranslateModule.forRoot(), ArtemisTestModule, TreeviewModule.forRoot(), RouterTestingModule.withRoutes([]), ArtemisSharedModule, FormDateTimePickerModule],
-            declarations: [LectureAttachmentsComponent],
+            declarations: [LectureAttachmentsComponent, MockPipe(HtmlForMarkdownPipe)],
```

Comment:
is the MockPipe(HtmlForMarkdownPipe) still needed although you have removed the pipe in the html-template? Maybe I am wrong, but I didn't see the pipe being added somewhere. 😅

Reply:
I removed the pipe from the Guided Tour template, it is still present in the lecture-attachment one.